README.md

```diff
@@ -0,0 +1,78 @@
+# mattermost-github-integration
+
+This role sets up a Mattermost GitHub integration service which posts changes of an organization or repository to a mattermost channel.
+For more information see [softdevteams repository](https://github.com/softdevteam/mattermost-github-integration)
+
+## Requirements
+
+This role requires an apt based system like Ubuntu.
+
+
+## Role Variables
+
+| Name                                           |         Required         | Default                              | Description                                                         |
+|:-----------------------------------------------|:------------------------:|:-------------------------------------|:--------------------------------------------------------------------|
+| `global_cache_dir`                             |    :heavy_check_mark:    |                                      | Cache directory to download roundcube files to                      |
+| `mattermost_github_integration_install_path`   | :heavy_multiplication_x: | `/opt/mattermost-github-integration` | Path to install the server on                                       |
+| `mattermost_github_integration_secret`         | :heavy_multiplication_x: | `None`                               | Secret that is used to authenticate between GitHub and your server. |
+| `mattermost_github_integration_server_hook`    | :heavy_multiplication_x: | `/`                                  | The relative URL where the server listens on.                       |
+| `mattermost_github_integration_server_address` | :heavy_multiplication_x: | `0.0.0.0`                            | Address of the server                                               |
+| `mattermost_github_integration_server_port`    | :heavy_multiplication_x: | `5000`                               | Port under which the server listens to webhooks from GitHub         |
+| `mattermost_github_integration_username`       | :heavy_multiplication_x: | `Github`                             | Username to post under in Mattermost                                |
+| `mattermost_github_integration_icon_url`       | :heavy_multiplication_x: | ` `                                  | URL to icon file which should show up in Mattermost                 |
+| `mattermost_github_integration_webhook_urls`   | :heavy_multiplication_x: | `[]`                                 | List of webhooks to post to. See below for more information         |
+| `mattermost_github_integration_show_avatars`   | :heavy_multiplication_x: | `true`                               | Show GitHub avatars in the message that is posted                   |
+| `mattermost_github_integration_ignore_actions` | :heavy_multiplication_x: | `[]`                                 | List of actions to ignore- See below for more information           |
```

Comment:
Dont you mean `ignore.` instead of `ignore-`?

Reply:
fixed